README.rst

```diff
@@ -16,30 +16,88 @@ utility for Python.
 
 The ``gcovr`` command can produce different kinds of coverage reports:
 
--  default or :option:`--txt<gcovr --txt>`: compact human-readable summaries
--  :option:`--html<gcovr --html>`: HTML summaries
--  :option:`--html-details<gcovr --html-details>`: HTML report with annotated source files
--  :option:`-x/--xml<gcovr --xml>`: machine readable XML reports in Cobertura_ format
--  :option:`--sonarqube<gcovr --sonarqube>`: machine readable XML reports in Sonarqube format
--  :option:`--json<gcovr --json>`: JSON report with source files structure and coverage
--  :option:`--json-summary<gcovr --json-summary>`: JSON summary coverage report
--  :option:`--csv<gcovr --csv>`: CSV report summarizing the coverage of each file
--  :option:`--coveralls<gcovr --coveralls>`: machine readable JSON reports in Coveralls_ format
+.. list-table::
+   :header-rows: 1
+
+   * - CLI Option
```

Comment:
Can the option be a link (removing the Guide column)?

Reply:
In the Sphinx version (doc/source/index.rst) the CLI Option column links to the CLI reference (e.g. `` :option:`--txt` ``), whereas the Guide column links to the longer output format documentation.

I agree that this looks a bit dumb in the README version, but RST doesn't make it possible to have a different number of columns in each version. There would also be difficulties around creating a link that contains ``` ``code`` ``` markup.

So I'd leave it as it is, but revisit this matter if a better design becomes apparent.

Reply:
<img width="676" alt="Bildschirmfoto 2022-01-11 um 21 02 17" src="https://user-images.githubusercontent.com/40258682/149012936-0c3f64b9-ed67-4270-a2cf-3710435a70ca.png">
The link ttext `docs` in each line is little bit confusing for me. What about using the headline of the format `Text Output`, `HTML Output`,`Coveralls JSON Output`, ... instead (Definition of abstract-guide-txt, ... in index.rst)?

Reply:
I thought about it and your suggestion makes sense. I'll update the table accordingly.